### DISCUSSION TITLE
Allow collaboration in Hedy programs like in Google Docs

### DISCUSSION THREAD

**Discussion by Felienne** [hedyorg/hedy#5094] [Large Projects & Plans]
Very cool, probably very hard!

**Comment by declann** [comment]
There is a blog post about building a realtime collaboration plugin for CodeMirror here:

https://currankelleher.medium.com/codemirror-6-experiments-a3930bf03781 (final section, with link to a GitHub [repo](https://github.com/vizhub-core/codemirror-ot))

It uses [ShareDB](https://github.com/share/sharedb).

I think this is POC of what Ross uses in [VizHub](https://vizhub.com/) and [vzcode](https://github.com/vizhub-core/vzcode).

He teaches visualization so I believe developed this for real-time assistance - I'm not sure how much demand there is for that, or live collaboration between students, in Hedy!
